src/Function/Properties/Equivalence.agda

```diff
@@ -0,0 +1,45 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- An Equivalence (on functions) is an Equivalence relation
+--   This file is meant to be imported qualified.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Function.Properties.Equivalence where
+
+open import Function.Bundles using (Equivalence; _⇔_)
+open import Level using (Level)
+open import Relation.Binary using (Setoid; IsEquivalence)
+open import Relation.Binary.PropositionalEquality using (setoid)
+
+import Function.Construct.Identity as Identity
+import Function.Construct.Symmetry as Symmetry
+import Function.Construct.Composition as Composition
+
+private
+  variable
+    a b ℓ₁ ℓ₂ : Level
+
+------------------------------------------------------------------------
+-- Setoid bundles
+
+module _ (R : Setoid a ℓ₁) (S : Setoid b ℓ₂) where
+
+  equivalence : IsEquivalence (Equivalence {a} {b})
```

Comment:
This should be named `isEquivalence` rather than `inverse` right? And I'd rename `⇔` to `⇔-isEquivalence`. Otherwise we're going to end up with a lot of name clashes! Same for the `Inverse` file below.

Reply:
Done.